docs/releases/5.2.md

```diff
@@ -183,3 +183,32 @@ If you are using the undocumented dropdown template tag with the offset arg, thi
 | Old                                                                | New                                                                  |
 | ------------------------------------------------------------------ | -------------------------------------------------------------------- |
 | `{% dropdown toggle_tippy_offset="[0, -2]" %}...{% enddropdown %}` | `{% dropdown toggle_tooltip_offset="[0, -2]" %}...{% enddropdown %}` |
+
+### Dropping support for `<script ...` templates
```

Comment:
I'd suggest changing this title to something that references `escape_script`:
```suggestion
### `escapescript` template tag and `escape_script` function are deprecated
```

I think this better describes the practical effect that the change in Wagtail will have on user code. We never really "supported" `<script>` templates - we just used them in our own code - and we don't really have a say in whether or not user code makes use of the same technique, except that now they can't use Wagtail's utilities for it.

Also, this way the majority of users - who aren't affected by this change - will easily be able to skim the release notes and see that this doesn't apply to them. I could easily see people reading the original title and thinking "what, we're not allowed to use `<script>` tags in our code any more?"

Reply:
Thank you, this is much better. I have updated accordingly.